users/admin.py

```diff
@@ -64,10 +64,10 @@ def save(self, commit=True):
         user.set_password(self.cleaned_data["password1"])
         user.email = self.cleaned_data["email"]
         if commit:
-            user.is_active = False # not active until he opens activation link
```

Comment:
ok, thanks.
BTW, the comment above this function is quite confusing, can you remove or modify it?